admin/class-admin-asset-manager.php

```diff
@@ -94,6 +94,20 @@ public function register_styles( $styles ) {
 	}
 
 	/**
+	 * A list of styles that shouldn't be registered but are needed in other locations in the plugin.
+	 *
+	 * @return array
+	 */
+	public function special_styles() {
+		return array(
+			'inside-editor' => new WPSEO_Admin_Asset( array(
+				'name' => 'inside-editor',
+				'src' => 'inside-editor-330',
```

Comment:
Is this file generated? If not it might be a problem using the version number in filenames.